### ISSUE TITLE
Crash when opening chatterino

### ISSUE THREAD

**Issue by kiwec** [Chatterino/chatterino2#2224]
**Describe the bug**
When opening chatterino, after connecting to the `vadikus007` channel, it crashes. I removed my twitch account from the accounts list because my session expired, but I doubt this is related.

**Chatterino version/Operating system**
`Chatterino 2.2.2 (commit )`  
That's the `chatterino2` package in the void linux repository.

**Additional information**
```
**Screenshots**
Helix::makeRequest called without a client ID set BabyRage
Helix::makeRequest called without an oauth token set BabyRage
[FFZEmotes] Reload FFZ Channel Emotes for channel "110240192"
Successfully listened to nonce "{9204bf0f-5a98-4a54-8312-edd3907c7eb7}"
Successfully listened to nonce "{bfd22089-2cc2-4806-861d-a06bd97ae96d}"
Successfully listened to nonce "{964de4fc-5bfc-4f8c-9387-b5d51964f44d}"
Successfully listened to nonce "{ba968bca-4033-4ad6-bfb2-53f6997a0006}"
Listen to topic "community-points-channel-v1.72256775"
[2020-11-25 09:27:14] [frame_header] Dispatching write containing 1 message(s) containing 8 header bytes and 141 payload bytes
[2020-11-25 09:27:14] [frame_header] Header Bytes: 
[0] (8) 81 FE 00 8D F2 7A 53 93 

Helix::makeRequest called without a client ID set BabyRage
Helix::makeRequest called without an oauth token set BabyRage
[FFZEmotes] Reload FFZ Channel Emotes for channel "72256775"
Successfully listened to nonce "{665d5a40-de97-4167-a087-a39b6439ad95}"
Successfully listened to nonce "{9e455883-c96b-4e36-8b71-115b103edd44}"
Successfully listened to nonce "{c9bad184-0eeb-4e6a-af42-58ee08e1c238}"
Successfully listened to nonce "{8fb01055-3858-48e0-b2f1-b7259429ece0}"
Successfully listened to nonce "{7f65bca5-047e-49ec-91c8-8ce3e43f33a5}"
Successfully listened to nonce "{5b608ff0-2634-4ef8-8bc4-36f9a524f6b1}"
chatterino: src/providers/ffz/FfzEmotes.cpp:22: chatterino::Url chatterino::{anonymous}::getEmoteLink(const QJsonObject&, const QString&): Assertion `emote.isString()' failed.
fish: “chatterino” terminated by signal SIGABRT (Abort)
```

FFZ API response : https://hastebin.com/raw/bifobajire

**Comment by Mm2PL** [comment]
This is a known bug, if you can please update to either `v2.2.2-fix` or `v2.2.3-beta2` (current nightly as of 2020-11-25).

**Comment by alazymeme** [comment]
I had no issues with my original build on windows, but crashed on linux. Such an interesting bug
